packages/i18n/README.md

```diff
@@ -37,38 +37,129 @@ npx pot-to-php languages/myplugin.pot languages/myplugin-translations.php text-d
+Jed-formatted JSON object shape.
+
+**Related**
+
+-   <http://messageformat.github.io/Jed/>
+
+**Parameters**
+
+-   **data** `?Object`: Locale data configuration.
+-   **domain** `?string`: Domain for which configuration applies.
+
+### sprintf
+
+[src/index.js#L159-L167](src/index.js#L159-L167)
+
+Returns a formatted string. If an error occurs in applying the format, the
+original format string is returned.
+
+**Related**
+
+-   <http://www.diveintojavascript.com/projects/javascript-sprintf>
+
+**Parameters**
+
+-   **format** `string`: The format of the string to generate.
+-   **args** `Array<string>`: Arguments to apply to the format.
+
+**Returns**
+
+`string`: The formatted string.
+
+### \_n
+
+[src/index.js#L125-L127](src/index.js#L125-L127)
+
+Translates and retrieves the singular or plural form based on the supplied
+number.
+
+**Related**
 
-See: https://developer.wordpress.org/reference/functions/__/
+-   <https://developer.wordpress.org/reference/functions/_n/>
 
-`_x( text: string, context: string, domain: string ): string`
+**Parameters**
+
+-   **single** `string`: The text to be used if the number is singular.
+-   **plural** `string`: The text to be used if the number is plural.
+-   **number** `number`: The number to compare against to use either the singular or plural form.
+-   **domain** `?string`: Domain to retrieve the translated text.
+
+**Returns**
+
+`string`: The translated singular or plural form.
+
+### \_nx
+
+[src/index.js#L144-L146](src/index.js#L144-L146)
+
+Translates and retrieves the singular or plural form based on the supplied
+number, with gettext context.
+
+**Related**
+
+-   <https://developer.wordpress.org/reference/functions/_nx/>
+
+**Parameters**
+
+-   **single** `string`: The text to be used if the number is singular.
+-   **plural** `string`: The text to be used if the number is plural.
+-   **number** `number`: The number to compare against to use either the singular or plural form.
+-   **context** `string`: Context information for the translators.
+-   **domain** `?string`: Domain to retrieve the translated text.
+
+**Returns**
+
+`string`: The translated singular or plural form.
+
+### \_x
+
+[src/index.js#L107-L109](src/index.js#L107-L109)
 
 Retrieve translated string with gettext context.
 
-See: https://developer.wordpress.org/reference/functions/_x/
+**Related**
+
+-   <https://developer.wordpress.org/reference/functions/_x/>
 
-`_n( single: string, plural: string, number: Number, domain: string ): string`
+**Parameters**
 
-Translates and retrieves the singular or plural form based on the supplied number.
+-   **text** `string`: Text to translate.
+-   **context** `string`: Context information for the translators.
+-   **domain** `?string`: Domain to retrieve the translated text.
 
-See: https://developer.wordpress.org/reference/functions/_n/
+**Returns**
+
+`string`: Translated context string without pipe.
+
+### \_\_
```

Comment:
Had you considered applying the code formatting in generating titles for the methods? At least for this particular function, it becomes difficult to discern in the preview result:

![image](https://user-images.githubusercontent.com/1779930/53887042-6c680200-3fef-11e9-8904-8815b3c80078.png)

Reply:
In this case, it's a noticeable improvement, indeed. Created https://github.com/WordPress/gutenberg/issues/14290 to keep track of this (and other potential formatting issues we detect). This should be applied in batch in a single PR to all packages, once the PRs at https://github.com/WordPress/gutenberg/issues/14227 land (to avoid -even more- rebasing nightmares).